tools/pipelines/build-build-tools.yml

```diff
@@ -24,7 +24,7 @@ parameters:
 - name: buildToolsVersionToInstall
   displayName: Fluid build tools version (default = installs version in repo)
   type: string
-  default: latest
+  default: "^0.6.0-110101"
```

Comment:
You see to be using different version for different pipelines. Please sync with @tylerbutler and align on the action before merging this PR.